ethcore/src/miner/miner.rs

```diff
@@ -865,6 +865,26 @@ impl miner::MinerService for Miner {
 		self.params.read().gas_range_target.0 / 5
 	}
 
+	fn set_minimal_gas_price(&self, new_price: U256) -> Result<bool, ()> {
+		if cfg!(feature = "price-info") {
+			warn!("Can't update fixed gas price while automatic gas calibration is enabled.");
```

Comment:
That's not 100% true, the fact that there is an option to set non-fixed `GasPricer` doesn't imply that it's actually set.
I think this `if` should rather be replaced with something like:
```
if let GasPricer::Fixed(mut val) = *self.gas_pricer.lock() {
    // update and recalibrate
   *val = new_price;
   // ...
} else {
  // Stringified error would be returned to RPC caller which imho would be nice.
  return Err("...");
}
```